packages/cli/src/Init.ts

```diff
@@ -119,6 +143,15 @@ export class Init implements Command {
 
   Set up a new Prisma project and specify MySQL as the datasource provider to use
     ${dim('$')} prisma init --datasource-provider mysql
+
+  Set up a new Prisma project and specify prisma-client-go as the generator provider to use
+    ${dim('$')} prisma init --generator-provider prisma-client-go
```

Comment:
```suggestion
  Set up a new Prisma project and specify 'go run github.com/steebchen/prisma-client-go' as the generator provider to use
    ${dim('$')} prisma init --generator-provider "go run github.com/steebchen/prisma-client-go"
```

as prisma-client-go would probably work but it's not recommended to do

cc @janpio

Reply:
🤷